src/generic/stage2_build_tape.h

```diff
@@ -50,7 +50,13 @@ struct unified_machine_addresses {
 class structural_iterator {
 public:
   really_inline structural_iterator(const uint8_t* _buf, size_t _len, const uint32_t *_structural_indexes, size_t next_structural_index)
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR.

Reply:
done